app/src/main/java/com/asfoundation/wallet/ui/balance/RestoreWalletPasswordPresenter.kt

```diff
@@ -50,15 +51,20 @@ class RestoreWalletPasswordPresenter(private val view: RestoreWalletPasswordView
         .observeOn(viewScheduler)
         .doOnNext { handleWalletModel(it) }
         .doOnError {
-          walletsEventSender.sendWalletCompleteRestoreEvent(WalletsAnalytics.STATUS_FAIL, it.message)
+          walletsEventSender.sendWalletCompleteRestoreEvent(WalletsAnalytics.STATUS_FAIL,
+              it.message)
         }
-        .subscribe())
+        .subscribe({}, {
```

Comment:
don't you prefer to have      
    view.hideAnimation()
          view.showError(RestoreErrorType.GENERIC)

in a doOnError and subscribe with only  it.printStackTrace()? it's more clean